webpack/assets/javascripts/react_app/pages/common/PageLayout/__snapshots__/PageLayout.test.js.snap

```diff
@@ -0,0 +1,94 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`render audits page 1`] = `
+<div
+  id="main"
+>
+  <div
+    id="content"
+  >
+    <div
+      data-notifications={null}
+      id="toast-notifications-container"
+    >
+      <Connect(ToastsList) />
+    </div>
+    <div
+      id="breadcrumb"
+    >
+      <Connect(BreadcrumbBar)
+        data={
+          Object {
+            "data": Object {
+              "breadcrumbItems": Array [
+                Object {
+                  "caption": "root",
+                  "url": "/some-url",
+                },
+                Object {
+                  "caption": "child with onClick",
+                  "onClick": [MockFunction],
+                },
+                Object {
+                  "caption": "active child",
+                },
+              ],
+              "isSwitchable": false,
+              "resource": Object {
+                "nameField": "name",
+                "resourceUrl": "some/url",
+                "switcherItemUrl": "some/url/:id",
+              },
+            },
+          }
+        }
+      />
+    </div>
+    <Row
+      bsClass="row"
+      componentClass="div"
+    >
+      <Col
+        bsClass="col"
+        className="title_filter"
+        componentClass="div"
+        md={6}
+      >
+        <div
+          id="search-bar"
+        >
+          <Connect(SearchBar)
+            data={
+              Object {
+                "data": Object {
+                  "autocomplete": Object {
+                    "searchQuery": null,
+                    "url": "model/auto_complete_search",
+                  },
+                  "bookmarks": Object {
+                    "canCreate": true,
+                    "documentationUrl": "http://www.theforeman.org/manuals/1.19/index.html#4.1.5Searching",
```

Comment:
This is an outdated URL.

Reply:
these are just fixtures, came from the searchbar props that i'm importing.
<img width="786" alt="screenshot 2018-12-04 11 57 06" src="https://user-images.githubusercontent.com/24938324/49433923-bc021c00-f7bb-11e8-9a1f-75bc5b881039.png">

maybe we should replace this.

Reply:
@ekohl this appears in the searchBar snap aswell
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/SearchBar/__tests__/__snapshots__/SearchBar.test.js.snap#L37

Reply:
Looks like this is a pattern we need to fix. I wonder if we also need to be aware of branding where a product might want to use different docs URLs. Another thing to note is that we usually use the bare `theforeman.org` without the `www` subdomain.

Reply:
I updated the SearchBar fixtures, no more urls :)